jte-kotlin/src/test/java/gg/jte/kotlin/TemplateEngineTest.java

```diff
@@ -764,6 +768,41 @@ void paramWithDefaultValue() {
         assertThat(output.toString()).isEqualTo("Your age is 10");
     }
 
+    @Test
```

Comment:
@casid @edward3h, I'm unsure if this is the best way/place to test it, but let me know what you think.

Reply:
Looks good, I'd say!